src/activeLighting.js

```diff
@@ -71,6 +71,21 @@ class ATL {
             default: false,
             type: Boolean,
         });
+        game.settings.register("ATL", "allowControlWithoutGm", {
+            name: "Allow Control without GM",
+            hint: "Allow users with this role and above to control ATL effects while the GM is not present (Requires reload)",
+            scope: "world",
+            config: true,
+            type: String,
+            choices: {
+                4: 'Game Master',
+                3: 'Assistant GM',
+                2: 'Trusted Player',
+                1: 'Player'
+            },
+            default: 4,
+            onChange: foundry.utils.debounce(() => window.location.reload(), 100)
```

Comment:
Let's use Foundry's new v10 feature to do the reload for us.
```suggestion
            requiresReload: true
```

Reply:
Fixed in latest commit.